validation/markets-validator.js

```diff
@@ -30,16 +33,21 @@ var MIN_LATITUDE = -90.0;
 var MAX_LONGITUDE = 180.0;
 var MIN_LONGITUDE = -180.0;
 
+var ACCEPTABLE_WARNINGS_COUNT = 2;
```

Comment:
I thought the name of the constant explains it enough but I will add a comment to make it clear.

Reply:
:heavy_check_mark: Added a comment in 57d30879ef00827287e0e4b6099527175d5ef6bf.

Reply:
Yeah ok, it would be better if it was possible to silence/ignore  a check for a certain city by having for example an annotation or variable which tracks the cities.

This seems to work, so maybe my suggestion is something for another PR.

Reply:
Yes, maybe that is possible. I personally hope that it is possible to sort out the issues with the remaining cities. @derhuerst already managed to resolve the issues for Hamburg and another city.